php/example_code/dynamodb/Tables_CRUD.php

```diff
@@ -60,36 +57,9 @@
 ####################################################################
 # Updating the table
 
-echo "# Updating the provisioned throughput of table $tableName.\n";
```

Comment:
You can still do the update, you just need to set the billing mode back to PROVISIONED. Otherwise, we lose this action entirely in the example.